.circleci/config.yml

```diff
@@ -17,15 +22,34 @@ jobs:
       - run:
           name: Show produced Docker images
           command: docker images
+      - run:
+          name: Login to Docker Hub
+          command: docker login --username ${DOCKER_USER} --password ${DOCKER_PASSWORD}
+      - run:
+          name: Deploy image to Docker Hub
+          command: make push
```

Comment:
Deployment should be done only in master and stable branches as it happens in `st2enterprise-dockerfiles`.

See an example: https://github.com/StackStorm/st2enterprise-dockerfiles/blob/master/.circleci/config.yml#L38-L47